www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -84,4 +90,284 @@ function borg_project_metrics_cron() {
+
+  // Path to images directory.
+  $images_path = backdrop_get_path('module', 'borg_project_metrics');
+
+  // Block content.
+  $metrics_content = <<<HTML
+    <div class="project-metrics">
+      <div class="users-metric">
+        <p class="users-img"> <img src="$images_path/images/users.svg" width="66" /> </p>
+        <p class="users-number"> $users </p>
+        <p class="users-label"> users </p>
+      </div>
+      <div class="commits-metric">
+        <p class="commits-img"> <img src="$images_path/images/commit.svg" width="66" /> </p>
+        <p class="commits-number"> $commits </p>
+        <p class="commits-label"> commits </p>
+      </div>
+      <div class="comments-metric">
+        <p class="comments-img"> <img src="$images_path/images/comments.svg" width="66" /> </p>
+        <p class="comments-number"> $comments </p>
+        <p class="comments-label"> comments </p>
+      </div>
+    </div>
+    <div class="join-the-fun">
+      <p>
+        Join the fun!
+        <a href="https://github/backdrop/backdrop-issues/issues">
+          backdrop/backdrop
+        </a>
+      </p>
+    </div>
+HTML;
+  switch ($delta) {
+    case 'backdrop_project_metrics':
+      $block['subject'] = NULL;
+      $block['content'] = $metrics_content;
+      return $block;
+  }
+}
+
+/**
+ * Helper function to retrieve metric form DB for backdrop_project_metrics block.
+ *
+ * @param string $type
+ * The type of metric to retrieve: {commits, comments, users}.
+ *
+ * @return array $metric
+ * An array of the db fields of the most recent $type metric.
+ */
+function _get_display_metric_from_db(string $type) {
+  $metric = db_select('borg_project_metrics', 'b')
+    ->fields('b', array('total_to_date', 'date'))
+    ->condition('type', $type)
+    ->orderBy('date', 'desc')
+    ->range(0, 1)
+    ->execute()
+    ->fetchAssoc();
+
+    return $metric;
+}
+
+/**
+ * Helper function to write the metrics to the database.
+ *
+ * @param string $type
+ * The $type: {users, commits, comments} to write to the DB.
+ *
+ * @param string $date
+ * The date to record for the $type metric.
+ *
+ * @param int $total_to_date
+ * The total to record for the $type metric.
+ */
+function _put_metrics_into_db(string $type, string $date, int $total_to_date) {
+  db_insert('borg_project_metrics')
+    ->fields(
+      array(
+        'type' => $type,
+        'date' => $date,
+        'total_to_date' => $total_to_date,
+      )
+    )
+    ->execute();
+}
+
+/**
+ * Helper function to get the comments from a
+ * /backdrop-contrib/$project/issues/comments.
+ *
+ * @param string $project
+ * The backdrop-contrib project to poll for comments.
+ *
+ * @param string $date
+ * The since paramater for the github api in the url.
+ *
+ * @return array $project_comment_metrics
+ * Returns an array of json objects each element is a paginated response from the
+ * github api.
+ */
+function _get_contrib_comments(string $project, string $date) {
```

Comment:
we should properly namespace these helper functions.  `borg_project_metrics_get_contrib_comments` or something.